### PR TITLE
fix(web): allow copying text in photo viewer

### LINKED ISSUE #8583: Copy keyboard shortcuts do not work on Web
### The bug

In both Firefox and Chrome, the "Ctrl+C" keyboard shortcut does not function to copy text. You can still copy by right clicking and selecting Copy. Having the Ctrl C shortcut is useful when working with filenames and GPS coordinates. Immich web UI should not preempt this keyboard combination, which is what I believe is currently happening, and can lead to user confusion.

### The OS that Immich Server is running on

Debian 12

### Version of Immich Server

v1.100.0

### Version of Immich Mobile App

v1.100.0

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
N/A
```

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by lukashass**
Closes #8583

### CODE REVIEW

File: web/src/lib/components/asset-viewer/photo-viewer.svelte [hunk 1]
```diff
@@ -130,8 +130,12 @@
     }
   });
 
-  const onCopyShortcut = () => {
-    if (window.getSelection()?.type === 'Range') {
+  const onCopyShortcut = async () => {
+    const selection = window.getSelection();
+    if (selection?.type === 'Range') {
+      if (selection.toString().length > 0) {
+        await copyToClipboard(selection.toString(), true);
```
Comment (bo0tzz) [inline web/src/lib/components/asset-viewer/photo-viewer.svelte:137]: Alternatively we can set `preventDefault = false` on the shortcut ([see here](https://github.com/immich-app/immich/blob/57d94bce6865d222ed56078224076a682017869e/web/src/lib/actions/shortcut.ts#L63-L65)), just return from this `if type == range`, and then call preventDefault after the if block.

Comment (bo0tzz) [reply web/src/lib/components/asset-viewer/photo-viewer.svelte:137]: (thus letting the browser just go to the default copy behaviour)

Comment (lukashass) [reply web/src/lib/components/asset-viewer/photo-viewer.svelte]: That is indeed better, didn't realize that `preventDefault` was true by default.

**Review by bo0tzz — APPROVED** [review]
Lovely, thank you!

**Review by jrasm91 — COMMENTED** [review]
Should this not be something that is implemented in the use:shortcuts implementation? Not anywhere/everywhere we might use a shortcut that has a default behavior? IMO all custom short cuts should be ignored if there is a current range selection, not just this one.

**Comment by mmomjian** [comment]
Thank you!

**Review by zackpollard — APPROVED** [review]

**Pull request merged**